src/registrar/utility/csv_export.py

```diff
@@ -267,10 +268,11 @@ def write_requests_csv(
                 # It indicates that DomainInformation.domain is None.
                 logger.error("csv_export -> Error when parsing row, domain was None")
                 continue
+        total_body_rows.extend(rows)
```

Comment:
**Question:** To confirm, you're essentially "adding" `rows` to another list and then after finishing the loop, we write it all at the very end, correct?

Reply:
Correct